infra/modules/gcp-psoxy-rest/main.tf

```diff
@@ -70,6 +70,8 @@ resource "google_cloudfunctions_function" "function" {
   entry_point                  = "co.worklytics.psoxy.Route"
   service_account_email        = var.service_account_email
   labels                       = var.default_labels
+  docker_registry              = "ARTIFACT_REGISTRY"
```

Comment:
Now, it seems that functions are deployed by default in artifact registry if they are new, so specifing here that for all of them.